### ISSUE TITLE
Buttons and accessibility on Android

### ISSUE THREAD

**Issue by satya164** [software-mansion/react-native-gesture-handler#468]
I don't seem to be able to interact with gesture handler buttons using the accessibility tools on Android.

When you enable TalkBack on Android, there are 2 issues with the gesture handler's button:
- The `accessibilityRole` doesn't seem to be applied. talkback doesn't announce it as a button
- It's not possible to interact with the button via double tap

In this snack, I added a button with `TouchableNativeFeedback` where these two work as I expect, and button from `GestureHandler` where they don't work:

https://snack.expo.io/@satya164/gesture-handler-button-accessibility

**Comment by satya164** [comment]
Fix:

```js
<RectButton
  onPress={() => alert('Clicked RectButton')}>
  <View accessible accessibilityRole="button">
    <Text>RectButton</Text>
  </View>
</RectButton>
```

Thanks @osdnk

**Comment by osdnk** [comment]
Thanks @adamczyk777

**Comment by kkafar** [comment]
Closing as following the docs should resolve the problem
